components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2393,10 +2397,18 @@ subroutine calc_ice_relaxation_timescale(rho,t,rhofaci,     &
    real(rtype), intent(out) :: epsi
    real(rtype), intent(inout) :: epsi_tot
 
+#ifdef SCREAM_CONFIG_IS_CMAKE
+    if (use_cxx) then
+      call ice_relaxation_timescale_f(rho,t,rhofaci,     &
+                                      f1pr05,f1pr14,dv,mu,sc,qitot_incld,nitot_incld, &
+                                      epsi,epsi_tot)
+       return
+    endif
+#endif
 
 
    if (qitot_incld.ge.qsmall .and. t.lt.zerodegc) then
-      epsi = ((f1pr05+f1pr14*sc**thrd*(rhofaci*rho/mu)**0.5_rtype)*2._rtype*pi* &
+      epsi = ((f1pr05+f1pr14*bfb_cbrt(sc)*bfb_sqrt(rhofaci*rho/mu))*2._rtype*pi* &
```

Comment:
It would be useful to make this F90 change a separate non-BFB PR. Then make the rest of these changes a second BFB PR.